### DISCUSSION TITLE
How to give access to helmfiles to external customers

### DISCUSSION THREAD

**Discussion by jattind** [helmfile/helmfile#1331] [Questions]
I use Harbor for my container registry and my helm charts and can give access to customers to my product. I have started to use helmfile to manage my cluster, but it would be nice if I can provide the same level of support for my customers. In the README it says distribute it via Git, S3, etc. Are there any plans for integrating this with Harbor so customers don't have to have access to multiple locations. Is there a better solution for distributing helmfiles? Appreciate any suggestions. Thanks

**Comment by yxxhero** [comment]
@jattind need more details. thanks.

**Reply by jattind** [reply]
Let's say I have several helm charts for deploying nginx and bunch of other pods. I have these and my containers in Harbor registry: https://goharbor.io/docs/1.10/working-with-projects/working-with-images/managing-helm-charts/
Normally I would give them access to Harbor registry and let them download helm charts so that they can deploy all the pods. Since Helmfile can manages the complete deployment of several pods and their state, I want to also give them Helmfile associated with this deployment, but there is no way to store them in Harbor.
